pages/common/julia.md

```diff
@@ -1,7 +1,7 @@
 # julia
 
 > A high-level, high-performance dynamic programming language for technical computing.
-> More information: <https://julialang.org>.
+> More information: <https://docs.julialang.org/en/v1/manual/getting-started/>.
```

Comment:
A better link would be https://docs.julialang.org/en/v1/stdlib/REPL/.

Reply:
But it doesn't help with any of the `julia`'s CLI subcommands (such as evaluation, parallelization, disabling fast math). That's why I went for this link instead.

Reply:
Good point. The link I suggested is more about the REPL module than the CLI interface. It's a pity that the URL is very generic, but that's a minor nitpick. Let's keep your link. 👍